modules/plotting/include/modules/plotting/utils/axisutils.h

```diff
@@ -43,28 +43,42 @@ class Mesh;
 
 namespace plot {
 
+IVW_MODULE_PLOTTING_API std::vector<double> getMajorTickPositions(const MajorTickSettings& ticks,
+                                                                  dvec2 range,
+                                                                  size_t maxTicks = 1000);
+IVW_MODULE_PLOTTING_API std::vector<double> getMinorTickPositions(
+    const MinorTickSettings& minorTicks, const MajorTickSettings& majorTicks, dvec2 range,
+    size_t maxTicks = 10000);
```

Comment:
The default value does not make sense unless rendered on a super high-res screen.
1000? That would be every four pixels on a 4K display. More than enough I think

Reply:
Its not a default its a upper limit, to avoid having it generate millions of ticks for to small deltas...

Reply:
I meant that the default maximum should not be more than 1000 :)
Looks like that is the case in the other places in the same file

Reply:
ahh, yes, had it at 10000 first, but changed to 1000... missed that one...